src/core/psi_container.jl

```diff
@@ -17,7 +17,7 @@ mutable struct PSIContainer
         settings::PSISettings,
         jump_model::Union{Nothing, JuMP.AbstractModel},
     )
-        #PSY.check_forecast_consistency(sys)
+        # PSY.check_forecast_consistency(sys)
```

Comment:
We can remove this line, this is no longer a PSY function.

Reply:
Resolved in c359a46.